00-project-documentation/community/Safeguarding.md

```diff
@@ -0,0 +1,75 @@
+# Safeguarding risks & mitigation strategies
+
+Collectively, the AutSPACEs team together with the larger community have identified a number of potential risks that participating in different AutSPACEs events that could particularly affect vulnerable adults (e.g. autistic and co-occuring conditions). Jointly we have developed a wide range of mitigation strategies to minimize those risks. 
+
+The team of AutSPACEs event organisers and faciliators (in particular Sophia Batchelor, Georgia Aitkenhead & Bastian Greshake Tzovaras) are responsible for the implementation of these actions.
```

Comment:
```suggestion
The team of AutSPACEs event organisers and facilitators (in particular Sophia Batchelor, Georgia Aitkenhead & Bastian Greshake Tzovaras) are responsible for the implementation of these actions.
```
Minor typo.

Reply:
It shows which parts I immediately wrote on GH and that aren't just copy & paste from the Turing's docx form 😂